### PR TITLE
Ws 1184c

### PR DESCRIPTION
**Pull request by darrowcoucla**
Update the Authentication

### CODE REVIEW

**Comment by darrowcoucla** [comment]
statsform.module, line 91 should be:

drupal_goto('statsform/form');

Thanks!

**Comment by z3cka** [comment]
there's actually 2 conflicts, what about this one on line 250?
```
<<<<<<< WS-1184c
          drupal_goto( "/app/statsform/login" );
=======
          drupal_goto( "/app/statsform/form" );
>>>>>>> master
```

File: www/sites/all/modules/custom/statsform/statsform.module [hunk 1]
```diff
@@ -232,7 +241,10 @@ function statsform_page_alter(&$page) {
           $page['sf_page_name'] = $sf_page_name;
         }
 
-        if( $query_parameters[reloginhereflag] == 1 ) {
+        // if the user chose Individual Login
+        if( $_SESSION['statsform']['sfUnitIndividualID'] == 'individual' ) {
+          drupal_goto( "/app/statsform/login" );
```
Comment (z3cka) [inline www/sites/all/modules/custom/statsform/statsform.module]: or is this right? with `drupal_goto( "/app/statsform/login" );`

Comment (darrowcoucla) [reply www/sites/all/modules/custom/statsform/statsform.module]: Yes, this is correct:

drupal_goto( "/app/statsform/login" );

The user chose Individual login and is directed to a shib protected response

Comment (z3cka) [reply www/sites/all/modules/custom/statsform/statsform.module]: cool. will re-roll tonight.

Comment (z3cka) [reply www/sites/all/modules/custom/statsform/statsform.module]: did re-roll and it turns out that I did it right the first time.

**Comment by z3cka** [comment]
deploying to `www-test` now

**Comment by z3cka** [comment]
@darrowcoucla `Copying of code from here to www-test2.library.ucla.edu is complete.`

**Pull request merged**